rmgpy/tools/isotopes.py

```diff
@@ -0,0 +1,852 @@
+    degeneracy values are detected. False if incorrect degeneracy values
+    exist.
+
+    This method assuumes an equilibrium distribution of compounds and finds
+    the fluxes created by the set of reactions (both forward and
+    reverse). It then checks that the fluxes of each compounds are proportional to their
+    symmetry numbers (since lower symmetry numbers have higher entropy and should have a
+    larger concentration.)
+
+    inputs:
+
+    reaction_isotopomer_list - a list of reactions that differ based on isotope placement
+    print_data - output the table of fluxes obtained. useful for debugging incorrect values
+    r_tol_small_flux - fraction of maximum flux to count as 'zero'
+    r_tol_deviation - allowable numerical deviation in answers
+
+    This method has a few datastructures it utilizes:
+
+    product_structures - a list of tuples, (index, isotopomer_structure), containing reactants and products
+    product_list - a pandas.DataFrame that sotres the fluxes and symmetry values
+    """
+
+    from rmgpy.kinetics.arrhenius import MultiArrhenius
+    def store_flux_info(species, flux, product_list,  product_structures):
+        """
+        input:
+            species - The desired species that you'd like to modify the flux value of
+            flux - the amount to modify the flux of the species
+            product_list - a list of current flux and symmetry values
+            product_structures - a list of unlabled structures un the reaction class
+
+        modifies the product list by either adding on new row with the species' structure, flux
+        symmetry ratio, etc. or adds the flux to the species' current row in `product_list`
+
+        returns: modified product_list
+        """
+        # find the corresponding unlabeled structure
+        for struc_index, product_structure in enumerate(product_structures):
+            if compare_isotopomers(product_structure, species):
+                structure_index = struc_index
+                break
+        # store product flux and symmetry info
+        for index in product_list.index:
+            spec = product_list.at[index,'product']
+            # if species already listed, add to its flux
+            if product_list.at[index,'product_struc_index'] == structure_index \
+                        and spec.isIsomorphic(species):
+                product_list.at[index,'flux'] += flux
+                return product_list
+        # add product to list
+        symmetry_ratio = product_structures[structure_index].getSymmetryNumber() / float(species.getSymmetryNumber())
+        return product_list.append({'product': species,
+                                            'flux': flux,
+                                            'product_struc_index': structure_index,
+                                            'symmetry_ratio': symmetry_ratio},
+                                           ignore_index=True)
```

Comment:
Align indentation?

Reply:
I think I fixed this. All the lines are not the same indention still because the last item is not within the dictionary. Each element now aligns with it's corresponding open bracket.